taichi/codegen/codegen_opengl.cpp

```diff
@@ -216,29 +328,29 @@ class KernelGen : public IRVisitor
     const auto bin_name = bin->raw_name();
     if (bin->op_type == BinaryOpType::floordiv) {
       if (is_integral(bin->element_type())) {
-        emit("const {} {} = int(floor({} / {}));", dt_name, bin_name, lhs_name,
+        emit("{} {} = int(floor({} / {}));", dt_name, bin_name, lhs_name,
```

Comment:
Is there a good reason to remove those `const`'s?

Reply:
Since all `tmpxx` won't be assigned one more time. No need to specify `const` to some while some are not :)

Reply:
> all tmpxx won't be assigned one more time

I would say that's exactly why we should mark them as `const` :-)